### PR TITLE
Check release branch

### PR DESCRIPTION
**Pull request by ochosi**

### CODE REVIEW

**Review by ondrejbudai — APPROVED** [review]
looks good! :)

File: release.py [hunk 1]
```diff
@@ -326,6 +326,14 @@ def create_release(args, api):
                              f"## CHANGES WITH {args.version}", False, False, None)
 
 
+def show_release_branches(args):
+    branches = run_command(['git','branch']).split("\n")
+    for branch in branches:
+        if f"release-{args.version}" in branch:
```
Comment (diaasami) [inline release.py:332]: shouldn't this be == ?
if yes, then I would actually suggest removing the for loop and using
```
if f"release-{args.version}" in branches:
    msg_error(...)
```

Comment (ochosi) [reply release.py:332]: I didn't want to worry about cleaning up the whitespace that is returned from `git branch`, but yeah I could do that :)

File: release.py [hunk 2]
```diff
@@ -326,6 +326,14 @@ def create_release(args, api):
                              f"## CHANGES WITH {args.version}", False, False, None)
 
 
+def show_release_branches(args):
+    branches = run_command(['git','branch']).split("\n")
```
Comment (diaasami) [inline release.py:330]: I think it's better to use split(), it uses whitespace including new lines to split, so it would both cleanup and split.

**Pull request merged**